source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -192,6 +193,14 @@ namespace PlayFab
                     {
                         // Add HTTP headers
                         SetPredefinedHeaders(reqContainer, hRequest);
+                        if(!setPredefinedHeadersFailed)
+                        {
+                            SetErrorInfo(reqContainer, "Error in attempting to add Default Headers with HRESULT: " + std::to_string(GetLastError()));
```

Comment:
>HRESULT: " + std::to_string(GetLastError() [](start = 104, length = 42)

the return of GetLastError isn't an HRESULT. It's a win32 system error code. You need to use HRESULT_FROM_WIN32 to convert to an HR

there's another line where you have this same issue below.